src/app/component/ProductActions/ProductActions.style.scss

```diff
@@ -177,20 +216,36 @@
         }
     }
 
+    &-AttributesWrapper {
+        transform: translate3d(0, 0, 0);
+        backface-visibility: hidden;
+        perspective: 1000px;
+
+        @include after-mobile {
+            width: 100%;
+            order: 10;
+        }
+
+        &.animate {
+            animation: shake 820ms .5ms cubic-bezier(.36,.07,.19,.97) both;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected single space after "," in a single-line function (function-comma-space-after)](https://app.codacy.com/manual/ScandiPWA/base-theme/pullRequest?prid=4874123)